File: Linked List/Java/GFG - COUNT NODES OF LINKED LIST.java
// GFG - Count nodes of linked list
// QUESTION LINK - https://www.geeksforgeeks.org/problems/count-nodes-of-linked-list/1

class Solution {
    public int getCount(Node head) {
        Node temp = head;
        int count = 0;
        while (temp != null) {
            count++;
            temp = temp.next;
        }
        return count;
    }
}

File: Linked List/Python/GFG - COUNT NODES OF LINKED LIST.py
# GFG - Count nodes of linked list
# QUESTION LINK - https://www.geeksforgeeks.org/problems/count-nodes-of-linked-list/1

class Solution:
    def getCount(self, head):
        temp = head
        count = 0
        while temp:
            count += 1
            temp = temp.next
        return count
